public/video-ui/src/components/FormFields/ItemPicker.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import KeywordPicker from '../utils/KeywordPicker';
+
+class ItemPicker extends React.Component {
```

Comment:
Yes, this makes sense. I guess its quite specific form component so its unlikely to be reused.